docs/sidebars.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
     default: [
         'introduction',
```

Comment:
changes here mean that the docs need to be deployed manually on the `documentation` branch

Reply:
I also noticed that secret managers are not visible in the docs because of that reason